monitor/reducers/parcelReducer.js

```diff
@@ -88,6 +90,7 @@ async function reduceLANDRegistry(event) {
         { owner: to.toLowerCase(), last_transferred_at },
         { id: parcelId }
       )
+      await Atlas.updateAsset(parcelId, ASSET_TYPES.parcel)
```

Comment:
I made a search all and `updateAsset` does not exist. Maybe you are trying to use. `upsertAsset`

So far, we have only parcels in the atlas, could we call it `updateParcel` if we need this func?